specification/communication/data-plane/Rooms/preview/2022-02-01-preview/communicationservicesrooms.json

```diff
@@ -495,69 +371,65 @@
     "RoomParticipant": {
       "description": "A participant of the room.",
       "required": [
-        "communicationIdentifier"
+        "rawId",
+        "role"
       ],
       "type": "object",
       "properties": {
-        "communicationIdentifier": {
```

Comment:
What's the reason for turning away from the `communicationIdentifier` (that's used by chat, callautomation, mediacomposition)?
When calling `GET /rooms/{roomId}/participants` from an SDK, how are you planning to provide strongly-typed `CommunicationIdentifier` models back to Contoso? 
You could possibly achieve that by [`CommunicationIdentifier.FromRawId(rawId);`](https://learn.microsoft.com/en-us/azure/communication-services/concepts/identifiers?pivots=programming-language-csharp) but it'd be good to agree on the approach and the strategy for maintaining the `CommunicationIdentifierSerializer`.

Reply:
@petrsvihlik I had advised to embrace raw ids in the service but then transform them to strongly-typed identifiers in the SDK. Getting rid of identifiers in the service allows for a much cleaner RESTful service without RPC-like POST actions and instead embracing PATCH with json+merge+patch. In the future raw id can also be used as a participant id in the /participants collection for single-item APIs.

I'm confident that we can come up with a good transformation in the SDK building on top of our new Identifier<-> RawId conversion APIs. Thoughts?

Reply:
@DominikMe agreed. Just wanted to make sure we're all on the same page. ;)